### DISCUSSION TITLE
[Feature]  Add haptic feedback to timeline

### DISCUSSION THREAD

**Discussion by l0nax** [immich-app/immich#5342] [Feature Request]
### The feature

Adding haptic feedback when scrolling through the timeline simplifies "locking" in the right month.

For example, if you scroll through the timeline in Google Photos, there is a short vibration for every month.

One of the things Google did very well with its Photos app is the haptic feedback and how the gallery is presented to the user.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#5344
